### PR TITLE
English site

### PR DESCRIPTION
**Pull request by vtemian**

### CODE REVIEW

**Comment by vtemian** [comment]
@robertdolca @palcu pls CR this one

**Comment by robertdxyz** [comment]
@palcu post review pls

File: src/components/home-english.jsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+                            {this.props.current.edition.name}
+                        </h5>
+                    </Col>
+                </Row>
+                <Row className="xsmall-spacing" />
+                <Row>
+                    <Col md={3} mdOffset={5}>
+                        <p className="right-button">
+                            <Link to="about" className="link link-secondary">
+                                About contest
```
Comment (palcu) [inline src/components/home-english.jsx:62]: About the contest.

File: src/components/home-english.jsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+export default React.createClass({
+  displayName: "Home",
+
+  componentDidMount() {
+    this.props.changeLanguage("en");
+  },
+
+  render() {
+    return <div className="home">
```
Comment (palcu) [inline src/components/home-english.jsx:33]: Pune aici `className="home english"`.

Apoi in `home.less`. Aliniaza butonul ala cu `About`.

File: src/components/home-english.jsx [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+                        <h2>National Contest of IT&amp;C projects</h2>
+                        <h5 className="tagline">
+                            Edition&nbsp;
+                            {this.props.current.edition.name}
+                        </h5>
+                    </Col>
+                </Row>
+                <Row className="xsmall-spacing" />
+                <Row>
+                    <Col md={3} mdOffset={5}>
```
Comment (palcu) [inline src/components/home-english.jsx:59]: Lasa doar un `xs={12}` si aliniaza la mijloc din CSS.

**Comment by palcu** [comment]
Post review done.

**Comment by vtemian** [comment]
Mai sunt parti in care nu s-a tradus și parti unde am dat cu google
translate. Poti sa verifici si traducerea?

On Thu, Jul 16, 2015 at 11:09 AM, Alex Palcuie notifications@github.com
wrote:

> Post review done.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/infoeducatie/infoeducatie-ui/pull/193#issuecomment-121872233
> .

## 

Vlad

**Pull request merged**